operations/external-db.yml

```diff
@@ -46,5 +46,115 @@
     user: ((terraform_outputs.bosh_rds_username))
     password: ((terraform_outputs.bosh_rds_password))
 
+- type: replace
+  path: /instance_groups/name=bosh/jobs/name=credhub/properties/credhub/data_storage
+  value:
+    type: postgres
+    host: ((terraform_outputs.credhub_rds_host))
+    port: ((terraform_outputs.credhub_rds_port))
+    username: ((terraform_outputs.credhub_rds_username))
+    password: ((terraform_outputs.credhub_rds_password))
+    database: credhub
+    require_tls: true
+
+- type: replace
+  path: /instance_groups/name=bosh/jobs/name=credhub/properties/credhub/tls?
```

Comment:
Should this be https://github.com/pivotal-cf/credhub-release/blob/master/jobs/credhub/spec#L193-L194 ?